com.leohanney.Volaris.yml

```diff
@@ -0,0 +1,38 @@
+id: com.leohanney.Volaris
+runtime: org.gnome.Platform
+runtime-version: '46'
+sdk: org.gnome.Sdk
+          
+command: volaris-gui
+rename-icon: com.leohanney.Volaris
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=network
+  - --share=ipc
+
+          
+modules:
+  - name: binary
+    buildsystem: simple
+    sources:
+      - type: file
+        url: https://github.com/volar-is/volaris-gui/releases/download/app-v0.1.0/Volaris_0.1.0_amd64.deb
+        sha256: c55febeee6cbde904b52119e86606dd08eb474aa8fc53be16843841334cdcc3c
```

Comment:
why not build from source ?

Reply:
This is the official method suggested by Tauri

Reply:
they are suggesting it wrong, if it's possible please build from source. We have multiple tauri apps building from source on Flathub

Reply:
> they are suggesting it wrong, if it's possible please build from source. We have multiple tauri apps building from source on Flathub

doing that now, thanks

Reply:
if you can't then revert to using the DEBs.

Reply:
> if you can't then revert to using the DEBs.

Alrighty, i'll stil try and get it to work in the background but having no luck with npm caches rn